hi-ml-histopathology/src/SSL/datamodules_and_datasets/dataset_cls_utils.py

```diff
@@ -0,0 +1,32 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from typing import Any, Tuple, Union
+
+import torch
+
+OptionalIndexInputAndLabel = Union[Tuple[torch.Tensor, int], Tuple[int, torch.Tensor, int]]
```

Comment:
are we using this anywhere?

Reply:
Yes it's the return type for __getitem__ in CxrDatasetBase